bot/cogs/snakes.py

```diff
@@ -29,6 +44,16 @@ def __init__(self, bot: AutoShardedBot):
         :return: A dict containing information on a snake
         """
 
+        url = f'https://en.wikipedia.org/w/api.php?action=query&titles={name}' \
+              f'&prop=extracts&exlimit=1&explaintext&format=json&formatversion=2'
+
+        # account for snakes without a page somewhere. >(PC)
+        async with ClientSession() as session:
+            async with session.get(url) as response:
+                resp = json.loads(str(await response.read(), 'utf-8'))
```

Comment:
Can't you just `return await response.json()`?

Reply:
I keep getting a TypeError when trying that, so I decided to build it using "json.load()". I still works exactly as I would like it to, without all the TypeError stuff.

"usable_data = web_data['query']['pages'][0]['extract']
TypeError: 'generator' object is not subscriptable"

I also completely rewrote it, so while this is still there, it's not returning anything until it's actually done doing what it's supposed to do. You'll see it in the next commit in a little while, or below if you're invested enough.

![image](https://user-images.githubusercontent.com/24819750/37874801-2b3acd72-3004-11e8-8eea-0639ec66bc27.png)